### ISSUE TITLE
lr_find() issue with NER

### ISSUE THREAD

**Issue by ga263** [amaiya/ktrain#109]
```
/usr/local/lib/python3.6/dist-packages/tensorflow_core/python/framework/tensor_shape.py in __init__(self, value)
    191       raise TypeError("Cannot convert %s to Dimension" % value)
    192     else:
--> 193       self._value = int(value)
    194       if (not isinstance(value, compat.bytes_or_text_types) and
    195           self._value != value):

TypeError: int() argument must be a string, a bytes-like object or a number, not 'tuple'
```

lr_find() breaks with the error above (last frame) when running with bilstm-crf and bilstm-bert NER models.

**Comment by amaiya** [comment]
Which version of TensorFlow are you using?  Another user reported that this problem was solved by upgrading from TF 2.0 to TF 2.1.0.  So, it seems to be a TensorFlow problem.

I was unable to reproduce this issue on TensorFlow 2.1.0.

Thanks.

**Comment by ga263** [comment]
I was using ```tensorflow_gpu==2.0.0```, so upgrading to 2.1.0 fixed the issue. Thanks Arun!
